src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/Completion/RazorDirectiveCompletionSourceTest.cs

```diff
@@ -108,7 +108,7 @@ public async Task GetDescriptionAsync_AddsDirectiveDescriptionIfPropertyExists()
 
             // Assert
             var description = Assert.IsType<string>(descriptionObject);
-            Assert.Equal(expectedDescription.Description, descriptionObject);
+            Assert.Equal(expectedDescription.Description, description);
```

Comment:
How did this pass before?

Reply:
`Assert.Equal` takes two objects.

Reply:
Yep, this is a code style change not a functional change.